src/node_buffer.cc

```diff
@@ -1271,6 +1271,11 @@ void Initialize(Local<Object> target,
               Integer::NewFromUnsigned(env->isolate(), kMaxLength)).FromJust();
 
   target->Set(env->context(),
+              FIXED_ONE_BYTE_STRING(env->isolate(), "kMaxStringLength"),
```

Comment:
Duplicate of `kStringMaxLength`.  Suggestion: rename that one and update the tests that reference it.